test/functional/passwords_controller_test.rb

```diff
@@ -28,9 +28,9 @@ class PasswordsControllerTest < ActionController::TestCase
       @user.forgot_password!
     end
 
-    context "with not found user" do
+    context "with incorrect token" do
       setup do
-        get :edit, params: { token: @user.confirmation_token, user_id: 0 }
+        get :edit, params: { token: @user.confirmation_token.succ }
```

Comment:
I feel the intent of this setup does not come across clearly without looking up what `succ` does. Can we use `"invalidtoken"` instead?

Reply:
Agreed. Thanks for the feedback.